src/openrct2/peep/Guest.cpp

```diff
@@ -2399,6 +2399,16 @@ bool Guest::HasRiddenRideType(int32_t rideType) const
     return RideTypesBeenOn[rideType / 8] & (1 << (rideType % 8));
 }
 
+void Guest::SetParkEntryTime(int32_t entryTime)
+{
+    ParkEntryTime = entryTime;
+}
+
+int32_t Guest::GetParkEntryTime() const
+{
+    return ParkEntryTime;
```

Comment:
We never use "passthrough getters" for fields that are public.

Reply:
The getter was requested in the issue, I'll remove it if we confirm that it's not wanted. I assume the "setter" should also be removed in that case?

Reply:
@ZehMatt suggested including it. I'm okay with it staying

Reply:
Its usually better to have getters so we can change the underlying data at any point without compatibility issues.

Reply:
> Its usually better to have getters so we can change the underlying data at any point without compatibility issues.

I second that

Reply:
Can't say I agree, but I'm not going to overrule three other team members for something like this.